modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -1,52 +1,67 @@
+
+////
+Metadata attribute that will help enable correct parsing and conversion to the appropriate DITA element.
+////
+
+:_mod-docs-content-type: PROCEDURE
+
 ////
 Base the file name and the ID on the module title. For example:
 * file name: proc_doing-procedure-a.adoc
 * ID: [id="doing-procedure-a_{context}"]
 * Title: = Doing procedure A
 
-Indicate the module type in one of the following
-ways:
-Add the prefix proc- or proc_ to the file name.
-Add the following attribute before the module ID:
-:_mod-docs-content-type: PROCEDURE
+The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 
-The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken. The `context` attribute enables module reuse. Every module ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in an assembly file.
```

Comment:
> Every module ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times *in an assembly file*.

Technically, this is not correct as using the same module multiple times in the same assembly file would give it the same context. It gets a unique ID if it is used multiple times in the same title and included in different assemblies. Including the same assembly in one title twice would result in the ID conflict too.

Reply:
Would you please suggest replacement wording? Does it enable cross-references?

Reply:
I like the new wording.